README.md

```diff
@@ -906,6 +907,20 @@ Web Development, React, Node.js, CSS, Next.js, Wordpress
 - [Github](https://github.com/mblayman)
 - [YouTube Playlist](https://www.youtube.com/watch?v=fNEZ_1dmVyE&list=PLFcKEo4b_n1wQA6lKtSqCq1dTMq1c2Lmw)
 
+### Mark Mandel
```

Comment:
This section needs to go above Matt Layman in order to remain alphabetically sorted. The line added in the index is already correct.